CoastalResilienceNetwork/regional-planning
# Remove Draw & Report feature

## Overview

Removes the "Draw & Report" feature per request from TNC.

Connects to https://github.com/CoastalResilienceNetwork/GeositeFramework/issues/1162

### Demo

![image](https://user-images.githubusercontent.com/1042475/51057463-af8b9300-15b3-11e9-8f1f-e3589282663e.png)

## Testing Instructions

- Add this plugin, with this branch checked out, to a local version of the framework.
- Interact with the plugin, and verify there are no errors related to draw and report.
